# request:
#   angles  float32[7]  the joint angles to move to (in radians)

float32[7] angles

---

# response:
#   success     bool    whether the request is successful (in which case the value field is valid)
#   message     string  description of request success/failure status

bool success
string message
